steveDevonport/steve_devonport.md

```diff
@@ -0,0 +1,46 @@
+
+When is a hero not a hero?
+
+In a previous role I was tasked with creating the Test Automation for a number of different web services, however when I joined the team I quickly discovered that they were lacking the capability to deliver customer value rapidly, securely, or with predictability. All this was compounded by the Scrum ceremonies not being orchestrated and there was no Scrum Master. The technology and tooling side was no better.
+
+After being convinced by the recruitment team and interview process that I would be working with an experienced and high performant group of people the above scenario came as a complete shock, whilst there might be a high performing group of people somewhere in the company...I guess I drew the short straw.
+
+As I saw it I had 2 choices...(1) quit and find a new role or (2) stick around and try to turn the team around.
+
+It's fair to say I chose option 2 otherwise you wouldn't be reading all about my experiences of working with a hero.
```

Comment:
why did you choose option 2? We'd like to hear why!

Reply:
I'll add a brief version behind my reasoning for choosing option 2.